includes/wikia/nirvana/WikiaView.class.php

```diff
@@ -204,7 +206,20 @@ private function getTemplateOptions( $controllerClass, $methodName, $controllerB
 		return $templates;
 	}
 
-	protected function getTemplateAnnotation( $controllerClass, $methodName ) {
+	private function getClassName( string $controllerClassWithNamespace ): string {
```

Comment:
What if we added this logic to `WikiaApp::getBaseName`? In that case we don't have to modify `WikiaView`.